tests/utils/tiny_model_summary.json

```diff
@@ -0,0 +1,3181 @@
+{
+    "AlbertModel": {
```

Comment:
There is nothing in this file that can't be directly infer from the auto mapping names classes. So it shouldn't be added IMO.

Reply:
This file contains the result of the tiny model creation, not the mapping we can infer from the auto mappings.
These are 2 different things. We can have a model class in a mapping, but we may not be able to create a tiny model (or tokenizer, processor) for it.

This file contains what have been created. So far only for tokenizer or processor, and not model classes (as I forgot to add it). If an object name is in this file, we should try to load it - if it fails, we should error out, not skip. If the name is not in this file, we should skip instead of error out.

But again, it's better for us to have a more detailed discussion.

Finally, a more complete comment
https://github.com/huggingface/transformers/pull/21516#discussion_r1109534311